### ISSUE TITLE
Won't run after python update

### ISSUE THREAD

**Issue by doorbuster** [kanehekili/VideoCut#35]
At least the python update to 3.12 is when I noticed that it no longer runs properly.  I'm on Arch (EndeavourOS).  Upon starting, I get an Unexpected Error dialog, with the following details:

```
Traceback (most recent call last):
*  File "/usr/bin/videocut", line 673, in __queueStarted
    self._videoController.prepare()
*  File "/usr/bin/videocut", line 1351, in prepare
    VideoPlugin.showBanner()
*  File "/opt/videocut/MpvPlayer.py", line 668, in showBanner
    self.initPlayer("icons/film-clapper.png",None)
*  File "/opt/videocut/MpvPlayer.py", line 550, in initPlayer
    raise Exception(self.player.lastError)
*Exception
```
Thanks

**Comment by kanehekili** [comment]
Yes, I checked it on Arch Linux - I'll have a new version ready soon - thanks for reporting

**Comment by kanehekili** [comment]
I have some news: Neither the python5 nor the upcoming python6 version is able to cope with the latest MPV library. Arch Linux has deployed libmpv.so.2.3.0, where the video widget will be displayed in its own window...
Currently I'm not able to determine if it is a glitch of the underlying libraries. Reverting mpv back to 1:0.37.0-2 would solve the issue in the meantime. I'll keep this open until a solution is found.

Update: currently I am negotiating with the mpv team. It seems to be an upstream error.

**Comment by Loganavter** [comment]
same trouble.

**Comment by kanehekili** [comment]
Released V3.0.0 on AUR with python QT6 and support for mpv 0.38. Closing this
